ietf/templates/doc/draft/change_state.html

```diff
@@ -12,9 +12,8 @@ <h1>
     </h1>
     {% if state.slug == "dead" %}
         <p class="alert alert-warning my-3">
-           This document is in IESG state "Dead". It is unusual to change
-           this to anything other than "AD is watching", and this should
-           never be used as a replacement for Begin IESG Processing.
+            This document is in IESG state "Dead". It is unusual to change this to
```

Comment:
Add back the "never be used as a replacement for Begin IESG Processing". (New IESG members assume that moving things into pubreq will have all the side-effects that the form/view for Begin IESG Processing applies, but that's not possible.

We could also consider not allowing the document to make any other transition in this particular case.